table.go

```diff
@@ -465,13 +465,15 @@ coalesceLoop:
 			}
 			itMore, err := it.onValue(dims, itVals)
 			if err != nil {
+				log.Errorf("Error while iterating: %v", err)
```

Comment:
@forkner I'm logging the error just in case something pops up here that we're not expecting.